### PR TITLE
Fix build: x86_64

### PR DESCRIPTION
**Pull request by BachToTheFuture**
Should build on x86_64. Don't know about x86 or x86_gcc2.
This is for an upcoming recipe :)

How I built/tested the source:
* There are 3 directories. Run `make` in each of them.
* Go to DebugMonitor and run the build. No windows should appear.
* Go to Scribble and run the build. Click on the main window. A DebugMonitor window should appear.
* Quit Scribble. There should be a new entry on the DebugMonitor window.

### CODE REVIEW

File: DebugClient/DebugClient.cpp [hunk 1]
```diff
@@ -4,10 +4,11 @@
  * $Log$
  *
  */
-
+#include <AppKit.h>
 #include "DebugClient.h"
 
-#include <Application.h>
+#include <InterfaceKit.h>
```
Comment (korli) [inline DebugClient/DebugClient.cpp]: Please don't use the Kit headers, only the ones for the classes actually needed.

File: DebugClient/DebugClient.cpp [hunk 2]
```diff
@@ -4,10 +4,11 @@
  * $Log$
  *
  */
-
+#include <AppKit.h>
 #include "DebugClient.h"
```
Comment (korli) [inline DebugClient/DebugClient.cpp:8]: "DebugClient.h" should come first (it should be self contained).

File: Scribble/ScribbleView.cpp [hunk 3]
```diff
@@ -1,15 +1,16 @@
 #include "ScribbleView.h"
 
 #include "../DebugClient/DebugClient.h"
-
+#include <AppKit.h>
+#include <SupportKit.h>
```
Comment (korli) [inline Scribble/ScribbleView.cpp]: Likewise, please use only headers for classes, not the kit ones.

File: DebugClient/DebugClient.cpp (hunk 1 above)
Comment (BachToTheFuture) [reply DebugClient/DebugClient.cpp]: @korli Will fix in a moment :)

File: DebugMonitor/DebugMonitor.h [hunk 4]
```diff
@@ -10,10 +10,13 @@
 
 #include "Tracer.h"
 #include <Application.h>
-#include <map.h>
+#include <SupportKit.h>
+#include <InterfaceKit.h>
```
Comment (korli) [inline DebugMonitor/DebugMonitor.h]: Likewise, please use only headers for classes, not the kit ones.

**Comment by BachToTheFuture** [comment]
@korli Done! Sorry for taking so long :)

**Review by korli — APPROVED** [review]

**Pull request merged**